### PR TITLE
#161966860 Hound CI Integration

### PR DESCRIPTION
**Pull request by leonardnjura**
#### What does this PR do?
Comments on code quality and style issues on pull requests

#### Description of Task to be completed?
Login to houndCI website with self or company Github repository and add it to https://www.houndci.com/repos

#### How should this be manually tested?
Clone the repository by running in your cmd
`$ git clone -b ch-integrate-hound-ci-161966860 --single-branch https://github.com/andela/ah-infinity-stones`

#### Any background context you want to provide?
Refer to linting configurations here http://help.houndci.com/configuration

#### What are the relevant pivotal tracker stories?
`161966860`

#### Questions:
Do you have any comments or suggestions on this PR?

### CODE REVIEW

**Comment by PromasterGuru** [comment]
@leonardnjura great work adding the Houd CI and also following the recommended syntax of creating a PR. Just try to make the pivotal tracker stories id to be a link, that is, to the story you were implementing.

**Comment by ThaDeveloper** [comment]
@leonardnjura Nice work. For shell commands, you can format them like so `$ git clone -b ch-integrate-hound-ci-161966860 --single-branch https://github.com/andela/ah-infinity-stones`.

**Comment by kamyaD** [comment]
@leonardnjura Great work and detailed description.

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+eslint:
+  version: 5.7.0
```
Comment (amutava) [inline .hound.yml]: add an end of line after end of file please!

Comment (leonardnjura) [reply .hound.yml]: Thanks for pointing that out @andela-amutava

**Pull request merged**